msteams-platform/resources/schema/manifest-schema.md

```diff
@@ -808,7 +808,7 @@ When `defaultBlockUntilAdminAction` property is set to **true**, the app is hidd
 
 **Maximum size** - 128 characters
 
-The `publisherDocsUrl` is an HTTPS URL to an information page for admins to get guidelines before allowing an app, which is blocked by default. It can also be used to provide any instructions or information about the app, which can be useful for the tenant admin.
+The value of the `publisherDocsUrl` parameter is a secure, HTTPS URL to the app documentation and information page that app developers choose to provide. Specifically, tenant admins get documentation about the app via this URL in the Teams admin center. The documentation may include the instructions for admins before they allow the app to facilitate app adoption and app rollout. The URL destination can also include instructions or information about the app that is useful for the tenant admin, users, and other business stakeholders.
```

Comment:
1. Please remove comma after 'The value of the `publisherDocsUrl` parameter is a secure,'

2. Please change the sentence to: 
    The tenant admins get documentation about the app using this URL in the Teams admin center.

Reply:
Thanks @v-preethah for the suggestions. Please verify the updates.

Reply:
Hi @v-preethah  and @surbhigupta12 , please verify the changes and publish this PR if all's good.